### ISSUE TITLE
Unused argument in correlation_threshold 

### ISSUE THREAD

**Issue by cells2numbers** [shntnu/pertminr#55]
The function 

```{r}
correlation_threshold <- function(population, variables, sample, cutoff = 0.90,
                                  method = "pearson") 
```

defines the argument population. In the following code, ```population``` is not used. 

Question: is the population argument used to have a more unique signature for all cytominer functions or can it be removed?

**Comment by shntnu** [comment]
You are right - it should be removed. Also true for `variance_threshold`

**Comment by cells2numbers** [comment]
Fixed with #58
